### PR TITLE
ADEN-4586 Facebook Audience Network in prebid.js POC

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/ADEN-4586

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/audienceNetwork.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+						[320, 50]
+					],
+					placementId: '963689110376230_1245839585494513'
+				}
+			}
+		};
+
+	function isEnabled() {
+		// TODO: add and set and check here instantGlobals.wgAdDriverAudienceNetworkBidderCountries
+		return adContext.getContext().targeting.skin == 'mercury';
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/audienceNetwork.js]: `===`

File: extensions/wikia/AdEngine/resources/prebid/prebid.js [hunk 2]
```diff
@@ -1764,6 +1764,8 @@
 	exports.registerBidAdapter(new VertozAdapter(), 'vertoz');
```
Comment (bognix) [inline extensions/wikia/AdEngine/resources/prebid/prebid.js]: minify please

File: extensions/wikia/AdEngine/js/lookup/prebid/adaptersRegistry.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
 define('ext.wikia.adEngine.lookup.prebid.adaptersRegistry', [
 	'ext.wikia.adEngine.lookup.prebid.adapters.appnexus',
-	'ext.wikia.adEngine.lookup.prebid.adapters.indexExchange'
-], function(appnexus, index) {
+	'ext.wikia.adEngine.lookup.prebid.adapters.indexExchange',
+	'ext.wikia.adEngine.lookup.prebid.adapters.audienceNetwork'
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/lookup/prebid/adaptersRegistry.js]: alphabetical order 🙊

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/audienceNetwork.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+						[300, 250],
+						[320, 50]
+					],
+					placementId: '963689110376230_1245839585494513'
+				}
+			}
+		};
+
+	function isEnabled() {
+		// TODO: add and set and check here instantGlobals.wgAdDriverAudienceNetworkBidderCountries
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/audienceNetwork.js:38]: create follow up story for production ready code and add this as a subtask / AC please

Comment (bognix) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/audienceNetwork.js:38]: and add FAN to `AdInfoTracker`

**Comment by bognix** [comment]
LGTM

**Review by bognix — APPROVED** [review]

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/audienceNetwork.js (hunk 4 above)
Comment (dianafa) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/audienceNetwork.js:38]: ticket created https://wikia-inc.atlassian.net/browse/ADEN-4638

**Pull request closed without merging**